tests/resource_filtering_test.go

```diff
@@ -1295,27 +1289,19 @@ var _ = Describe("Resource includes", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Veryfing backup")
-				backup, err := framework.GetBackup(timeout, backupName, f.BackupNamespace)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(backup.Status.Progress.ItemsBackedUp).To(Equal(backup.Status.Progress.TotalItems))
-
-				// The backup should contains the following 8 items:
-				// - DataVolume CRD
-				// - DataVolume
-				// - PVC
-				// - PV
-				// - VolumeSnapshot
-				// - VolumeSnapshotContent
-				// - VolumeSpapshotClass
-				// - Namespace
-				expectedItems := 8
-				if framework.IsDataVolumeGC(f.KvClient) {
-					expectedItems -= 1
+				expectedItems := map[string][]string{
+					"CustomResourceDefinition": []string{"datavolumes"},
+					"DataVolume":               []string{includedDVName},
```

Comment:
If a DataVolume exists, the DataVolume CRD must exist so don't think we have to explicitly check